honeybadger/fake_connection.py

```diff
@@ -0,0 +1,7 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+def send_notice(config, payload):
+    logger.debug('Using a fake connection. Will not make a real call to Honeybadger API')
```

Comment:
It might make sense to log this line on the info level, since a user will always want to know that the error isn't actually being reported in development. What do you think?

Reply:
Yeah I think is completely fair. To be honest I wasn't sure if we want it to add any other behaviour apart from logging, so that is why I started with debug level, but I think it makes complete sense what you are saying 👍